### ISSUE TITLE
High percentage of CPU usage in ssdlite_mobilenet model 

### ISSUE THREAD

**Issue by masoudnick** [tensorflow/tensorflow#35483]
Hi all,
I trained a model for license plate detection with ssdlite_mobilenet_v2_coco (training images:20000,steps:200000) and this model is going to be used in a traffic control app.

The accuracy of this model on frames of video is very good but the percentage of CPU usage on my system is between 70-80% (CPU info : Intel Core i7-6500 @ 2.50GHz.) and given that traffic control app should be constantly running on client system, this percentage is quite high.

So what are the best ways to optimize CPU usage to analyze frames of video and detect license plate? (reduce to ~30-40%)
Do I need to make any changes to the config file or use another pre-trained model?
Please guide me :)

**Comment by ymodak** [comment]
You may try optimizing your model using quantization, or use quantized mobilenet v2 model.
See https://www.tensorflow.org/lite/performance/best_practices#optimize_your_model
